### ISSUE TITLE
A way for dangerfile to detect whether is running on CI

### ISSUE THREAD

**Issue by stellarhoof** [danger/danger-js#811]
We've got rules that make sense to only run when on CI. Currently there's no way of detecting this without duplicating the logic danger already uses to detect if it's running in CI.

When I say CI I mean real CI, not the fake one. Perhaps this information could be exposed under `danger.utils` ?

**Comment by orta** [comment]
Yeah, that makes sense `danger.utils.info.isCI` maybe? Then we can add something like `isGitHub`, or `isLocal` later on

**Comment by stellarhoof** [comment]
I'd say `danger.utils.info.ci.*`, so we've got a nice namespace there.

**Comment by orta** [comment]
I'm cool with that - it means someone doesn't have to feel blocked on implementing because we agree on the general idea 👍

**Comment by milesj** [comment]
https://www.npmjs.com/package/env-ci ?

**Comment by orta** [comment]
Yep good point, you're welcome to use that in your Dangerfiles - I'd rather not have a new dependency in Danger unless it provides something really new and useu
